synapse/federation/sender/per_destination_queue.py

```diff
@@ -352,18 +352,18 @@ async def _transaction_transmission_loop(self) -> None:
 
                 self._start_catching_up()
         except FederationDeniedError as e:
-            logger.info(e)
+            logger.info(e.msg)
         except HttpResponseException as e:
             logger.warning(
-                "TX [%s] Received %d response to transaction: %s",
+                "TX [%s] Received %d response to transaction:",
                 self._destination,
                 e.code,
-                e,
+                exc_info=e,
```

Comment:
So this is the sort of place where it really doesn't make much sense to include a stack trace. as its very common for remote servers to return non-200 responses. 

Plus, the stack trace doesn't tell us anything we don't already know at this stage.